thirdparty/spdlog-20170710/tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#

```

Comment:
Should remove the tests subdirectory and skip this part of the build for the lib